app/main.py

```diff
@@ -1,3 +1,78 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __add__(self, other: int) -> int:
```

Comment:
The return type annotation for the `__add__` method should be `Distance` instead of `int`, as it returns a `Distance` object.